src/coreclr/jit/emitarm64.cpp

```diff
@@ -22654,6 +22889,21 @@ void emitter::emitDispLowPredicateRegPair(regNumber reg, insOpts opt)
     printf(" }, ");
 }
 
+//------------------------------------------------------------------------
+// emitDispVectorRegPair: Display a pair of vector registers
+//
+void emitter::emitDispVectorRegPair(regNumber reg, insOpts opt)
```

Comment:
can we reuse `emitDispSveConsecutiveRegList` instead?

Reply:
Yes we can; updated.